### ISSUE TITLE
Offer a way to move requests between stagings

### ISSUE THREAD

**Issue by coolo** [openSUSE/open-build-service#7360]
Offer a way (an API will do) to move a request between 2 stagings without going through the backlog.

- Remove package from A
- Add unselect history to A
- Add package to B
- add select history to B
- add review by_project B
- accept review by_project A

**Comment by hennevogel** [comment]
What keeps you from doing this with the existing APIs? Accepting the review?

**Comment by coolo** [comment]
Not sure what you're asking. There is no API to move between stagings, only to move to or from backlog

**Comment by hennevogel** [comment]
Well there is an API to unstage from A, there is an API to stage into B. Both add history and handle reviews.

**Comment by coolo** [comment]
Right. and this feature request is about 'a way without going through backlog', i.e. without email notifcation to staging managers that there is a new review
